ddtrace/contrib/internal/graphql/patch.py

```diff
@@ -325,6 +351,12 @@ def _set_span_errors(errors: List[GraphQLError], span: Span) -> None:
             path = ",".join([str(path_obj) for path_obj in error.path])
             attributes["path"] = path
 
+        error_extension_fields = config.graphql._error_extensions
+        if error_extension_fields is not None:
+            extensions = _validate_error_extensions(error, error_extension_fields)
+            if extensions:
+                attributes["extensions"] = extensions
```

Comment:
The attribute key here should be `attributes[f"extensions.{key}"] = value_for_key`, instead of a nested value. This is necessary because we can't store a dictionary as a Span Event value. For example:
```python
attributes={
  message="Cannot query field \"__typenam\" on type \"Query\".",
  locations=["1,2"],
  path=["query myQuery", "__typenam"]
  extensions.code="downstream_client_error",
  extensions.status=400,
  extensions.response='{"message": "Unauthorized"}',
}
```